README.md

```diff
@@ -1,7 +1,13 @@
 # joint-military-symbology-xml
 Joint Military Symbology XML (JointMilSyML or JMSML) is an XML schema, and associated instance data, designed to definitively document the contents of MIL-STD 2525D and NATO STANAG APP-6(C).
 
-___Check out this [Symbology Explorer](http://kjellmf.github.io/military-symbology-explorer/#/home) written by @kjellmf . It's a perfect example of how open source development of the JMSML benefits everyone!___
+## Projects using this Repo
+
+|Project|Repository|Website|
```

Comment:
Can you move this under the resources section & add the 2 other project names/cells "milsymbol" & "joint military symbology explorer"

& it should be good to merge.